From 541e397ee645dfa7783eef3cc2c4cfc345cace30 Mon Sep 17 00:00:00 2001
From: Jim Jones <rauchwolke@gmx.net>
Date: Wed, 20 Apr 2022 16:18:20 +0000
Subject: [PATCH 1/3] Fix osd dialog position

After the upgrade to kde-frameworks-5.93 the osd volume indicator is displayed in the top left corner of the screen instead of being displayed in the center of the screen.

BUG: 452648
(cherry picked from commit aec00b32980390b93411f140dfb0503a671c2163)
---
 src/plasmaquick/dialog.cpp | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/plasmaquick/dialog.cpp b/src/plasmaquick/dialog.cpp
index 09937ef0f..478a8d404 100644
--- a/src/plasmaquick/dialog.cpp
+++ b/src/plasmaquick/dialog.cpp
@@ -1237,11 +1237,7 @@ bool Dialog::event(QEvent *event)
 {
     if (event->type() == QEvent::Expose) {
         if (!KWindowSystem::isPlatformWayland() || !isExposed()) {
-            auto ret = QQuickWindow::event(event);
-            if (d->mainItem) {
-                d->syncToMainItemSize();
-            }
-            return ret;
+            return QQuickWindow::event(event);
         }
 
         /*
-- 
2.35.1

